debug_toolbar/utils.py

```diff
@@ -229,14 +229,17 @@ def getframeinfo(frame, context=1):
         return (filename, lineno, frame.f_code.co_name, lines, index)
 
 
-def get_sorted_request_variable(variable):
+def get_sorted_request_variable(variable, *, source):
     """
     Get a sorted list of variables from the request data.
     """
-    if isinstance(variable, dict):
-        return [(k, variable.get(k)) for k in sorted(variable)]
-    else:
-        return [(k, variable.getlist(k)) for k in sorted(variable)]
+    try:
+        if isinstance(variable, dict):
+            return [(k, variable.get(k)) for k in sorted(variable)]
+        else:
+            return [(k, variable.getlist(k)) for k in sorted(variable)]
+    except TypeError:
+        return [(f"<{source}>", variable)]
```

Comment:
Since the wrapping call already has this associated with the attribute on the request (get, post, files), I don't think we need to use the source. I think using an empty string or something constant will be more appropriate.

Reply:
Thanks, that's a good point. I have pushed an update but I haven't tested the update (yet)